src/main/resources/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM/help-credentialsId.html

```diff
@@ -1,13 +1,12 @@
 <div>
-    <p>When running a job, Jenkins will use these credentials to check out the source code for builds. For Jenkins to do
-        this, it needs credentials with access to the projects and repositories you want it to build from.</p>
-
-    <p>To provide Jenkins with credentials you can:</p>
+    <p>When running a job, Jenkins requires credentials to checkout the source code for builds. To do this, it needs credentials with access to the projects and repositories you want it to build from.</p>
+    <p>You can provide Jenkins with credentials here by:</p>
     <ul>
-        <li>choose credentials from the list</li>
-        <li>add credentials as a <strong>Username with password</strong> (for the password, you can enter a Bitbucket Server password or a
+        <li>selecting credentials from the list</li>
+        <li>adding credentials as a <strong>Username with password</strong> (for the password, you can enter a Bitbucket Server password or a
             Bitbucket Server <a href="https://confluence.atlassian.com/x/a97-Nw">personal access token</a>)
         </li>
-        <li>Leave this empty to use default credentials if they exist.</li>
+        <li>leaving this empty to use default credentials if they exist</li>
```

Comment:
so.. @mhenschke-atl  these default credentials do not actually work, right? and you hope to remove them very soon. Maybe we could remove this line now that we're here editing anyway?

Reply:
Yes, I think so. This is _technically_ incorrect, given this is a known bug and we don't intend on fixing it. May be better to remove the message entirely from the three help files.